qa/tasks/mgr/dashboard/test_feedback.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+from .helper import DashboardTestCase
+
+
+class FeedbackTest(DashboardTestCase):
+
+    def test_create_api_key(self):
+        self._ceph_cmd(['mgr', 'module', 'enable', 'feedback'])
+
+        time.sleep(10)
+        self._post('/api/feedback/api_key', {'api_key': 'testapikey'})
+        self.assertStatus(201)
```

Comment:
```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls._ceph_cmd(['mgr', 'module', 'enable', 'feedback'])
        time.sleep(10)

    @classmethod
    def tearDownClass(cls):
        super().tearDownClass()
        cls._ceph_cmd(['mgr', 'module', 'disable', 'feedback'])

    def test_create_api_key(self):
        self._post('/api/feedback/api_key', {'api_key': 'testapikey'})
        self.assertStatus(201)
```

Reply:
FYI: the tearDown implementation is not strictly necessary, just in case we want to cover the module disabling.